apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -117,11 +117,14 @@ therein; a full rebuild is forced whenever the command has changed (e.g.
 Optional arguments:
   -h, --help            Show this help message and exit.
   -p COUNT, --parallelism COUNT
-                        Specify the number of concurrent build processes The 
-                        value \\"max\\" can be specified to indicate the number 
-                        of CPU cores. If this parameter omitted, the default 
-                        value depends on the operating system and number of 
-                        CPU cores.
+                        Specifies the maximum number of concurrent processes 
+                        to launch during a build. The COUNT should be a 
+                        positive integer or else the word \\"max\\" (which 
+                        specifies a count equal to the number of CPU cores). 
+                        If this parameter omitted, the default value depends 
+                        on the operating system and number of CPU cores. This 
+                        parameter may alternatively specified via the 
```

Comment:
This parameter may alternatively "be" specified

Reply:
Thanks, fixed